packages/cli/lib/index.ts

```diff
@@ -36,30 +42,35 @@ const program = new NangoCommand();
 dotenv.config();
 
 program.name('nango').description(
-    `By default, the CLI assumes that you are using Nango Cloud so you need to set the NANGO_SECRET_KEY env variable or pass in the --secret-key flag with each command.
+    `By default, the CLI assumes that you are using Nango Cloud so you need to set the NANGO_SECRET_KEY env variable.
```

Comment:
See comment left on the overall PR.

Reply:
Updated with https://github.com/NangoHQ/nango/pull/760/commits/c3fbb79ba1d6c812997931e2492358b3ea58507d